packages/automerge-repo-react-hooks/test/useHandle.test.tsx

```diff
@@ -6,11 +7,14 @@ import {
 } from "@automerge/automerge-repo"
 import { DummyStorageAdapter } from "@automerge/automerge-repo/test/helpers/DummyStorageAdapter"
 import { render, waitFor } from "@testing-library/react"
-import React from "react"
 import { describe, expect, it, vi } from "vitest"
 import { useHandle } from "../src/useHandle"
 import { RepoContext } from "../src/useRepo"
 
+import { next as A } from "@automerge/automerge"
+// Do something with 'next' so that it isn't compiled away
+const someDoc = A.init()
```

Comment:
Uhhhh, we should talk about this!

Reply:
I have a solution! Unfortunately the solution (which involves subpath exports on the `@automerge/automerge-repo` package) has broken use of things like

```typescript
import { eventPromise } from "@automerge/automerge-repo/src/helpers/eventPromise"
```

which appears in lots of the tests . This means I need to find a proper place to put all of these helpers. Once I've figured that reorganization out I'll push an update.